modules/appeals_api/app/controllers/appeals_api/metadata_controller.rb

```diff
@@ -98,5 +90,38 @@ def upstream_service_details(service_name, health_checker, time)
         }
       }
     end
+
+    def decision_reviews_versions
+      if Settings.modules_appeals_api.documentation.path_enabled_flag
+        [
+          decision_reviews_v1.merge({ status: VERSION_STATUS[:previous] }),
+          decision_reviews_v2
+        ]
+      else
+        [
+          decision_reviews_v1
+        ]
+      end
+    end
+
+    def decision_reviews_v1
+      {
+        version: '1.0.0',
+        internal_only: true,
+        status: VERSION_STATUS[:current],
```

Comment:
im guessing one of these (probably this one?) should be marked `previous` instead of `current`?

Reply:
ah, i just noticed it overrides the status above on line 97...
hmmmm, i dont personally love this (just because it wasnt immediately obvious to me) but i do get why you did it.
whatever works 👍🏻

Reply:
i almost think it would be better, imo, if this just also checked that setting down here inline.
would mean checking the feature flag in 2 places, but would read better to me personally i think

Reply:
hmm... i'm not a huge fan of putting conditionals inside hashes. Unless you meant something else?

Reply:
@SlakrHakr We're going to do away with the merge anyway as per our PM, so resolving this here